htdocs/admin/system/dbtable.php

```diff
@@ -52,10 +52,10 @@
 		while ($i < $num) {
 			$row = $db->fetch_row($resql);
 			if ($row[0] == $field) {
-				$sql = 'ALTER TABLE '.$db->sanitize($table).' MODIFY '.$db->sanitize($row[0]).' '.$row[1].' CHARACTER SET utf8';		// We must not sanitize the $row[1]
+				$sql = 'ALTER TABLE '.$db->sanitize($table).' MODIFY '.$db->sanitize($row[0]).' '.$db->sanitize($row[1]).' CHARACTER SET utf8';		// We must not sanitize the $row[1]
```

Comment:
@JonBendtsen have you seen the comment `// We must not sanitize the $row[1]`

Reply:
> @JonBendtsen have you seen the comment `// We must not sanitize the $row[1]`

No I did not see that, I just wanted my other pull requests to build. I guess I can just close this pull request then.

Reply:
let's wait and see what the coder wanted to do

Reply:
https://github.com/Dolibarr/dolibarr/issues/27998